cmd/enduro-a3m-worker/main.go

```diff
@@ -56,6 +57,13 @@ func main() {
 		fmt.Printf("Failed to read configuration: %v\n", err)
 		os.Exit(1)
 	}
+	// Set Defaults for the a3m and global task queues
+	if reflect.ValueOf(cfg.A3m.TaskQueue).IsZero() {
+		cfg.A3m.TaskQueue = temporal.A3mWorkerTaskQueue
+	}
+	if reflect.ValueOf(cfg.Temporal.TaskQueue).IsZero() {
+		cfg.A3m.TaskQueue = temporal.GlobalTaskQueue
+	}
```

Comment:
I haven't had time to review this PR before going on vacation, I just have one comment here: using the reflect package to check the zero value is probably overkill and slower. The idiomatic way is: `if x == "" { ... }`.

Reply:
If the config now contains the task queues I don't think we need to maintain `temporal.A3mWorkerTaskQueue` nor `temporal.GloblaTaskQueue`. You should probably use Viper to set defaults or have the application fail because these configuration attributes are mandatory and not allowed to be undefined.

Reply:
defaults are now set by viper. I did keep the values there just instead of raw strings because it makes it easy to track things down via lsp